shub/utils.py

```diff
@@ -669,9 +669,20 @@ def create_scrapinghub_yml_wizard(conf, target='default', image=False):
     Ask user for project ID, ensure they have access to that project, and save
     it to given ``target`` in local ``scrapinghub.yml`` if desired.
     """
+    if target.isdigit():
```

Comment:
Moving all the wizard logic into a helper function is a good step and I really like it. However the helper keeps growing and becomes too "branchy" (12 IFs in one function is a lot), it's hard to read, probably we could refactor it a bit and split to a few parts, wdyt?

I can clearly split it to 4 steps:
- checking if using wizard is required at all
- getting project id as input parameter (if needed)
- getting image repository as input parameter (if needed)
- updating local config file

Also it's possible that project is `None` and image is `False` [here](https://github.com/scrapinghub/shub/blob/ad3c8a97612e02877ce86c014f73ffd26036d426/shub/utils.py#L730), isn't it? I wonder if we should stop wizard if there're no changes.